### PR TITLE
adding implementable northside enounters

### PR DESCRIPTION
**Pull request by ThomasLipp**

### CODE REVIEW

File: eldritch/encounters.py [hunk 1]
```diff
@@ -244,8 +244,97 @@ def Science7(char):
   helping = events.PassFail(char, check, close_gates, fail)
   return events.BinaryChoice(char, "Offer to help?", "Yes", "No", helping, events.Nothing())
 
+def Shoppe1(char):
```
Comment (fingoldfish) [inline eldritch/encounters.py:247]: Note: the curiositie shoppe is named "Shop". Þe old magick shoppe is named "Shoppe". If that's confusing, you can feel free to rename them.

File: eldritch/test_encounters.py [hunk 2]
```diff
@@ -1158,12 +1158,144 @@ def testSquare7Pass(self):
     self.assertEqual(len(self.char.possessions), 1)
     self.assertEqual(self.char.possessions[0].name, "Cross")
     
+class ShoppeTest(EncounterTest):
+
+  def setUp(self):
+    super(ShoppeTest, self).setUp()
+    self.char.place = self.state.places["Shoppe"]
```
Comment (fingoldfish) [inline eldritch/test_encounters.py:1165]: Again, Shoppe -> Shop.

**Review by fingoldfish — APPROVED** [review]

File: eldritch/encounters.py (hunk 1 above)
Comment (ThomasLipp) [reply eldritch/encounters.py:247]: ah, okay, will rename everything

Comment (ThomasLipp) [reply eldritch/encounters.py]: everything has been renamed.

**Pull request merged**